tests/testthat/test-infix_spaces_linter.R

```diff
@@ -223,3 +214,19 @@ test_that("parse tags are accepted by exclude_operators", {
   expect_lint(text, list(col_assign, col_sub), infix_spaces_linter(exclude_operators = "EQ_FORMALS"))
   expect_lint(text, list(col_formals, col_sub), infix_spaces_linter(exclude_operators = "EQ_ASSIGN"))
 })
+
+test_that("lints vectorize", {
+  lint_msg <- rex::rex("Put spaces around all infix operators.")
+
+  expect_lint(
+    trim_some("{
+      a<-1
+      1/2
+    }"),
+    list(
+      list(lint_msg, line_number = 2L),
+      list(lint_msg, line_number = 3L)
+    ),
+    infix_spaces_linter()
```

Comment:
Shouldn't we also include a test for the case where there are multiple assignments on the same line?
Also, shouldn't there be two lints here?

``` r
library(lintr)

lint(
  text = "{
      a    <-1; b  <- 2
    }",
  linters = infix_spaces_linter()
)
#> <text>:2:12: style: [infix_spaces_linter] Put spaces around all infix operators.
#>       a    <-1; b  <- 2
#>            ^~
```

<sup>Created on 2023-12-07 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
> Also, shouldn't there be two lints here?

No, because `allow_multiple_spaces=TRUE` by default. That's to allow spacing to serve as alignment like

```r
a   = 1
aa  = 2
aaa = 3
```

Setting `allow_multiple_spaces=FALSE` gives the second lint.

Reply:
> Shouldn't we also include a test for the case where there are multiple assignments on the same line?

Sure, can't hurt; adding one now.

Reply:
Please check if the extended test fits what you had in mind.